src/test_l1_hdmi_cec_driver.c

```diff
@@ -1404,7 +1405,7 @@ void test_hdmicec_hal_l1_hdmiCecTx_sinkDevice_positive( void )
     int len = 2;
     //Get CEC Version. return expected is opcode: CEC Version :43 9E 05
     //Sender as 3 and broadcast
-    unsigned char buf[] = {0x3F, CEC_GET_CEC_VERSION};
+    unsigned char buf[] = {0x38, CEC_GET_CEC_VERSION};
```

Comment:
why 38 here?  F is the broadcast..

Reply:
Yes , This is to change the transmit from broadcast to specific address.
In broadcast we don't have an acknowledgement. hence will never get HDMI_CEC_IO_SENT_BUT_NOT_ACKD as a response and test will fail.

So to verify transmit and check the response changes from broadcast address to device address